src/main/scala/cromwell/engine/CallActor.scala

```diff
@@ -187,7 +187,8 @@ class CallActor(key: CallKey, locallyQualifiedInputs: CallInputs, backend: Backe
     )
 
     val message = executionResult match {
-      case SuccessfulExecution(outputs, returnCode, hash) => WorkflowActor.CallCompleted(key, outputs, returnCode, hash)
+      case SuccessfulExecution(outputs, returnCode, hash) =>
+        WorkflowActor.CallCompleted(key, outputs, returnCode, if (workflowDescriptor.writeToCache) hash else "")
```

Comment:
I'd love to have a tech talk regarding trivially wrapping primitive types in case classes... I'm still not convinced that every primitive in our system needs to be wrapped a case class.  These case classes, like `SymbolHash` can still lie to you.  The compiler might say "I wanted a `SymbolHash` and you gave me a `SymbolHash`, it's provably correct!"  But really since there's no guarantee that `SymbolHash` is actually a hash of a symbol, the caller should not make that assumption as well.  So the caller might get a `SymbolHash` but it should still treat it as an opaque string.

In this case, `SymbolHash` is just a different way to spell `String`.  So what's really being gained?

Now if `SymbolHash` also had more information, like the symbol it's a hash of and if it could only be constructed in a correct state, then that'd be a different story.

Anyway, I'd love to talk about it in detail.